README.md

```diff
@@ -65,7 +65,7 @@ The minimum amount of configuration for a wizard step is the `next` property to
 * `fields` - specifies which of the fields from the field definition list are applied to this step. Form inputs which are not named on this list will not be processed. Default: `[]`
 * `template` - Specifies the template to render for GET requests to this step. Defaults to the route (without trailing slash)
 * `backLink` - Specifies the location of the step previous to this one. If not specified then an algorithm is applied which checks the previously visited steps which have the current step set as `next`.
-* `controller` - The constructor for the controller to be used for this step's request handling. The default is an extension of the [hof-form-controller](https://www.npmjs.com/package/hof-form-controller), which is exported as a `Controller` property of this module. If custom behaviour is required for a particular form step then custom extensions can be defined - see [Custom Controllers](#custom-controllers)
+* `behaviours` - A single behaviour, or an array of behaviours to be mixed into the base controller to extend functionality.
```

Comment:
Is the order of behaviours significant when passed as an array? My guess is yes, so I'd probably want to know if they are going to be applied left-to-right or right-to-left.